### PR TITLE
Use create-react-app for the web build

### PR DESCRIPTION
**Pull request by kevinrobinson**
This tries reorganizing the files so that we can use `create-react-app` to build instead of maintaining our own build config.

Tested manually, and confirmed that test runners work as well.

### CODE REVIEW

**Comment by kevinrobinson** [comment]
@keving17 Can you try this out on your computer too?  May need to `rm -rf` your node_modules.

File: README.md [hunk 1]
```diff
@@ -21,8 +26,22 @@ $ npm run android
...
+You can run these in Native or web environments.
+
+```
+$ npm run web-test
+$ npm run native-test
+```
+
+
+### How does this work?
+It uses `create-react-native-app` to make the app, then `create-react-app` to make a web build, and `react-native-web` to make the React Native APIs work on the web.
```
Comment (keving17) [inline README.md]: Looks great!

File: web/webpack.config.js [hunk 2]
```diff
@@ -1,37 +0,0 @@
...
-                }
-            }
-        ]
-    },
-    resolve: {
-        alias: {
-            "react-native": "react-native-web"
-        }
-    }
-};
```
Comment (keving17) [inline web/webpack.config.js]: I'm loving this new webpack config file

**Comment by keving17** [comment]
I'll be sure to try out your changes tomorrow before merging this commit!

**Comment by kevinrobinson** [comment]
@keving17 Sounds good, thanks for reviewing!  👍   Yeah if this looks good to you locally, then go ahead and merge.  If not, comment and then we can figure it out Wednesday.

**Comment by kevinrobinson** [comment]
@keving17 awesome, thanks for fixing that up 👍

**Pull request merged**